hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -2160,3 +2156,113 @@ def hpat_pandas_series_median_impl(self, axis=None, skipna=True, level=None, num
         return numpy.median(self._data)
 
     return hpat_pandas_series_median_impl
+
+@overload_method(SeriesType, 'pct_change')
+def hpat_pandas_series_pct_change(self, periods=1, fill_method='pad', limit=None, freq=None):
+    """
+    Pandas Series method :meth:`pandas.Series.pct_change` implementation.
+
+    Note: Unsupported mixed numeric and string data
+
+    .. only:: developer
+
+       Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_pct_change
+       Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_pct_change_str
```

Comment:
I propose to list all tests via pattern:
`Test: python -m hpat.runtests -k hpat.tests.test_series.TestSeries.test_series_pct_change*`